python/ray/runtime_env/runtime_env.py

```diff
@@ -389,6 +389,11 @@ def __init__(
                     f"Specified fields: {invalid_keys}"
                 )
 
+            logger.warning(
+                "The `container` runtime environment field is DEPRECATED and will be "
+                "removed in the near future. Please use `image_uri` instead."
```

Comment:
maybe link to docs

Reply:
Also "near future" is too vague, how about we put a concrete date here and say "will be removed after July 31, 2025"?

Reply:
done and done, thanks!